CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.0 
+* Expose `style.localizeLabels`. 
+* Expose `mapboxMap.attribution`, `mapboxMap.logo`, `mapboxMap.compass` and `mapboxMap.scaleBar` settings.
+* Fix deployment target for iOS to 11.
```

Comment:
nit:
maybe divide into 3 block:
```
common changes
android
ios
```

it will be easy to find improvements and bug fixes

Reply:
Nice, will do